src/Icicle/Source/ToCore/Prim.hs

```diff
@@ -197,6 +197,15 @@ convertPrim p ann resT xts
   gofun DaysEpoch
    = return $ primmin $ Min.PrimDateTime Min.PrimDateTimeDaysEpoch
 
+  gofun Seq
```

Comment:
you should be able to convert this without adding a new primitive to Core. in fact having a Core primitive for it doesn't really make sense.
should be able to do something like:

| [ (xa, ta) , (xb, tb) ] <- xts
, getPossibilityOrDefinitely a == PossibilityPossibly
…
= generate
   Case xa
    | Left err -> Left err
    | Right _ ->
        (if tb is Possibly
         then Right xb
         else xb) 
    end